controllers/users.js

```diff
@@ -607,6 +607,25 @@ const setInDiscordScript = async (req, res) => {
   }
 };
 
+const removeTokens = async (req, res) => {
+  try {
+    const users = await userQuery.fetchUsersWithToken();
+
+    if (!users.length) {
+      return res.status(404).json({ message: "No users found with github Token!" });
+    }
+
+    await userQuery.removeGitHubToken(users);
+
+    return res.status(200).json({
+      message: "Github Token removed from all users!",
+      usersFound: users.length,
+    });
```

Comment:
We can return a summary of the migration in the response.
Like adding the total users data were migrated and failed number of users for which the data could not be updated if such a case happens.

Reply:
@shubham-y as we are using batch for updates, it will be all or nothing , so i dont know if it will make sense to give metrics for failed updates